blueprints/component-test/index.js

```diff
@@ -25,17 +26,46 @@ module.exports = useTestFrameworkDetector({
   ],
 
   fileMapTokens: function() {
-    return {
-      __testType__: function(options) {
-        return options.locals.testType || 'integration';
-      },
-      __path__: function(options) {
-        if (options.pod) {
-          return path.join(options.podPath, options.locals.path, options.dasherizedModuleName);
+    if (isModuleUnificationProject(this.project)) {
+      return {
+        __test__() {
+          return 'component-test';
+        },
+        __root__(options) {
+          if (options.inRepoAddon) {
+            return path.join('packages', options.inRepoAddon, 'src');
+          }
+          return 'src';
+        },
+        __testType__(options) {
+          if (options.locals.testType === 'unit') {
+            throw 'The --unit flag isn\'t supported within a module unification app';
+          }
+          return '';
+        },
+        __path__(options) {
+          if (options.pod) {
+            throw 'Pods aren\'t supported within a module unification app';
```

Comment:
Please change to 
```js
throw new Error('...');
```
so we get a nice stack trace

Reply:
Sure, I will change that. I just wonder why this is like that in the original blueprints, I just copied from there...

Reply:
Plenty of stuff in the official repos could stand to be improved ;)

Reply:
Fixed

Reply:
@simonihmig PR to fix the pattern you copied from the original blueprints https://github.com/emberjs/ember.js/pull/16962